### PR TITLE
Update Node.js details

### PR DESCRIPTION
**Pull request by jasnell**
#### Summary
Updates various Node.js Web API compatibilities

### CODE REVIEW

File: api/BroadcastChannel.json [hunk 1]
```diff
@@ -32,6 +32,9 @@
           "ie": {
             "version_added": false
           },
+          "nodejs": {
+            "version_added": "15.4.0",
```
Comment (lucacasonato) [inline api/BroadcastChannel.json:36]: I suggest you add a note that this needs to be imported from "worker_threads" and that it is experimental. That would align with how api.MessageChannel was documented prior to being available as a global.

**Review by lucacasonato — COMMENTED** [review]
Ooh nice! Didn't know Node has WHATWG streams now. Such a great win for compat. I do think it is worth adding `"notes"` that the interfaces need to imported from "streams/web" + that they are marked as experimental in Node docs.

File: api/Blob.json [hunk 2]
```diff
@@ -26,6 +26,10 @@
           "ie": {
             "version_added": "10"
           },
+          "nodejs": {
+            "version_added": "15.7.0",
+            "notes": "Experimental implementation. Must be imported using require('buffer').Blob or import { Blob } from 'buffer'."
```
Comment (ddbeck) [inline api/Blob.json:31]: It's not true everywhere, but often structure this information with `"alternative_name"`.

Additionally, notes can be an array. I'd suggest two separate notes here, for consistency with similar cases:

- `"Experimental implementation."`
- `"Must be imported from the <code>buffer</code> module."`

```suggestion
            "version_added": "15.7.0",
            "alternative_name": "buffer.Blob",
            "notes": "Experimental implementation. Must be imported using require('buffer').Blob or import { Blob } from 'buffer'."
```

**Review by ddbeck — CHANGES_REQUESTED** [review]
Thanks for this PR, @jasnell. Also, I see it's your first to the project—welcome!

I've got a few consistency suggestions in line comments, but no major concerns heree. Let me know if there are any questions. Thanks again!

**Comment by ddbeck** [comment]
Hi @jasnell, sorry for the delay on the original review. Let me know if you plan to come back to this. If not, we can take over getting these changes merged. Thank you!

**Comment by jasnell** [comment]
Yes, sorry, I do want to get this going! I missed the questions that were there.

**Review by ddbeck — APPROVED** [review]
Looks good. Thank you! 🎉

**Pull request merged**